music.rb

```diff
@@ -0,0 +1,12 @@
+class MusicAlbum < Item
+  attr_accessor :on_spotify
+
+  def initialize(publish_date, on_spotify)
+    super(id, publish_date)
+    @on_spotify = on_spotify
+  end
+
+  def can_be_archived?
```

Comment:
Please make this method private according to the UML class diagram.
![image](https://user-images.githubusercontent.com/104029494/196652700-28d0e73f-a0b9-44a4-ad56-54acde2ff1b4.png)